modules/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/virt-using-skip-node.adoc
+//
+
+[id="virt-using-skip-node_{context}"]
+= Using skip node annotation
+
+* Use skip-node annotation to prevent the `node-labeller` from reconciling a node.
```

Comment:
This sentence seems like the intro sentence for the module (modules also need intro sentences). The bullet point is used to format the instructional step in a one-step procedure.